orca-echo/src/main/groovy/com/netflix/spinnaker/orca/echo/pipeline/ManualJudgmentStage.groovy

```diff
@@ -72,21 +76,34 @@ class ManualJudgmentStage implements StageDefinitionBuilder, AuthenticatedStage
     final long backoffPeriod = 15000
     final long timeout = TimeUnit.DAYS.toMillis(3)
 
-    @Autowired(required = false)
-    EchoService echoService
+    private final EchoService echoService
+
+    private final FiatPermissionEvaluator fiatPermissionEvaluator
+
+    WaitForManualJudgmentTask(Optional<EchoService> echoService, Optional<FiatPermissionEvaluator> fpe) {
```

Comment:
No `@Autowired` annotation needed on this constructor?

Reply:
There is no autowired annotation on this constructor. Please check the file again.

Reply:
I realize there isn't, but this class was changed from field injection to constructor injection and for consistency we usually add an `@Autowired` annotation to the constructor.

Reply:
Most of the project is going to have `@Autowired` on `@Component` constructors. I believe it works in this case because the class has only one constructor, but the norm as far as I'm concerned is to annotate the constructors even if there is only one.

This is a minor point on this PR.

Reply:
Should i use @Autowired or not? Since, this is a minor point. I am dropping this feedback.

Reply:
I asked you to remove the `@Autowired` properties in favor of a [single constructor](https://github.com/spinnaker/orca/pull/3947/files#r513595076). My example did not include an `@Autowired` annotation, but that wasn't really intended to be copy/pasted, either. While `@Autowired` on a single constructor is optional, per @ajordens, precedent in the rest of the codebase does have us adding `@Autowired` on single constructors anyway, probably for explicitness.

This is a good example for why keeping a single PR for a change is valuable: I had to click through 4 different PRs to find the original thread of feedback about this. Had it been a single PR, context for other reviewers would've been easily discoverable.

Reply:
done..added @Autowired